components/ast/src/parser_state.rs

```diff
@@ -21,7 +21,7 @@ impl ParserState {
         parser::parse(Cow::Borrowed(&*input_text), &mut module_table, 0)
     }
 
-    crate fn untern_string(&self, string_id: StringId) -> Arc<String> {
+    pub fn untern_string(&self, string_id: StringId) -> Arc<String> {
```

Comment:
This was meant to be used through the `HasParserState` trait, not directly.

Reply:
(Though this string interning system is probably intended to be removed eventually anyway...)